### PR TITLE
fix(kit): correct handling the `thousandSeparator` for InputRange

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

### CODE REVIEW

**Review by waterplea — APPROVED** [review]

**Comment by waterplea** [comment]
Are these formatted values used somewhere other than in template? Maybe we can use format number pipe?

**Comment by splincode** [comment]
`Are these formatted values used somewhere other than in template? Maybe we can use format number pipe?` 
no, we can’t, there are calculations inside the mask lambda function

File: projects/kit/abstract/input-slider.ts [hunk 1]
```diff
@@ -125,6 +126,9 @@ export abstract class AbstractTuiInputSlider<T>
     mask: TuiMapper<number, TuiTextMaskOptions> = (quantum: number, min: number) => ({
         mask: tuiCreateNumberMask({
             allowNegative: min < 0,
+            autoCorrectDecimalSymbol: !TUI_DECIMAL_SYMBOLS.includes(
```
Comment (vladimirpotekhin) [inline projects/kit/abstract/input-slider.ts:129]: nit: there is todo in tuiCreateNumberMask about deleting autocorrectDecimalSymbol in 3.0, so if it makes sense now can you delete todo plz?

**Review by vladimirpotekhin — APPROVED** [review]

File: projects/kit/abstract/input-slider.ts (hunk 1 above)
Comment (splincode) [reply projects/kit/abstract/input-slider.ts]: fixed

**Comment by splincode** [comment]
@vladimirpotekhin please, see again

File: projects/core/utils/mask/enable-auto-correct-decimal-symbol-by.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import {TUI_DECIMAL_SYMBOLS} from '@taiga-ui/core/constants';
+import {NumberFormatSettings} from '@taiga-ui/core/interfaces';
+
+export function tuiEnableAutoCorrectDecimalSymbolBy(
```
Comment (waterplea) [inline projects/core/utils/mask/enable-auto-correct-decimal-symbol-by.ts:4]: `By` is unnecessary

File: projects/core/utils/mask/enable-auto-correct-decimal-symbol-by.ts [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+    settings: NumberFormatSettings,
```
Comment (waterplea) [inline projects/core/utils/mask/enable-auto-correct-decimal-symbol-by.ts:5]: Destruct :)

File: projects/core/utils/mask/enable-auto-correct-decimal-symbol-by.ts (hunk 2 above)
Comment (splincode) [reply projects/core/utils/mask/enable-auto-correct-decimal-symbol-by.ts]: fixed

File: projects/core/utils/mask/enable-auto-correct-decimal-symbol-by.ts (hunk 3 above)
Comment (splincode) [reply projects/core/utils/mask/enable-auto-correct-decimal-symbol-by.ts]: fixed

**Pull request merged**